app/assets/javascripts/app/components/accordion.js

```diff
@@ -48,29 +49,41 @@ class Accordion {
     }
   }
 
+  handleKeyUp(event) {
+    const keyCode = event.keyCode || event.which;
+
+    if (keyCode === 13 || keyCode === 32) {
+      this.handleClick();
+    }
+  }
+
+  on(event, callback) {
+    this.el.addEventListener(event, callback);
+  }
+
+  emitEvent(target = null, eventType) {
+    const emittable = new Event(eventType);
+    (target || this.el).dispatchEvent(emittable);
+  }
+
+  setExpanded(bool) {
+    this.headerControl.setAttribute('aria-expanded', bool);
+  }
+
   open() {
-    this.el.setAttribute('aria-expanded', 'true');
-    this.content.setAttribute('aria-hidden', 'false');
+    this.setExpanded(true);
     this.content.classList.remove('display-none');
     this.content.classList.remove('animate-out');
     this.content.classList.add('animate-in');
+    this.emitEvent(this.el, 'accordion.show');
   }
 
   close() {
-    this.el.setAttribute('aria-expanded', 'false');
-    this.content.setAttribute('aria-hidden', 'true');
+    this.setExpanded(false);
```

Comment:
Would it be ok if we add `this.headerControl.focus();` within this close function so that if someone closes with the bottom close link, the focus outline gets reapplied to the header?

Like this:
![screen recording 2017-03-07 at 04 52 pm](https://cloud.githubusercontent.com/assets/1178494/23679707/c55693f4-0356-11e7-9845-245e188cd68c.gif)